src/MetalPerformanceShaders/MPSKernel.cs

```diff
@@ -320,6 +320,58 @@ public MPSImageLaplacianPyramidAdd (IMTLDevice device, nuint kernelWidth, nuint
 			}
 		}
 	}
+
+	public partial class MPSCnnBinaryConvolutionNode {
+		[TV (11,3), Mac (10,13,4, onlyOn64: true), iOS (11,3)]
+		public static MPSCnnBinaryConvolutionNode Create (MPSNNImageNode sourceNode, IMPSCnnConvolutionDataSource weights, float [] outputBiasTerms, float [] outputScaleTerms, float [] inputBiasTerms, float [] inputScaleTerms, MPSCnnBinaryConvolutionType type, MPSCnnBinaryConvolutionFlags flags)
+		{
+			unsafe {
```

Comment:
Can we validate the size of the arrays before passing pointers to native code?

Reply:
I don’t think there is an easy way to check for parameters sizes on those methods, they can even be omitted according to docs.